### ISSUE TITLE
[MU4 Issue] Last note of passage before rest cut off in Muse Strings

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#14078]
**Describe the bug**
In fast passages, a final note before a rest gets cut off, when using any of the string section sounds in Muse Sounds

**To Reproduce**
1. New score for Violins (section)
2. Set to Muse Sounds
3. Enter 4 sixteenth notes: C D E F
4. Play
5. Hear error: the F is only barely sounded, if it is sounded at all

**Expected behavior**
All notes played full duration

**Screenshots**
![image](https://user-images.githubusercontent.com/1799936/198131513-4142e86d-658e-4719-80e8-4cc7e7c6bd9c.png)

**Platform information**
 - OS: Linux

**Additional context**
Same for other strings, also a number of woodwinds (flute, oboe, clarinet, etc), but brass seem OK, at those I tried.

**Comment by Zanctarian** [comment]
Likely similar bug to #13970 
I experience this issue as well, especially with faster tempos.

Maybe issue #14051 is related to this bug?

**Comment by Tantacrul** [comment]
Yes. We have a few reports of this and probably have too many issues all pointing at the same problem. Once we get an update to beta out next week, we'll assess where we're at.

**Comment by MarcSabatella** [comment]
It seems better in the current beta I think, but still not really good.  The following passage in particular for violins 1 is where I initial discovered it, and it still chops off the last note (unless this fix is not part of the beta):

![image](https://user-images.githubusercontent.com/1799936/200075259-18293571-fe26-491c-a9c7-1f831f43f358.png)
